lib/jsdom/living/events/PointerEvent.webidl

```diff
@@ -0,0 +1,28 @@
+// https://www.w3.org/TR/pointerevents/#pointerevent-interface
```

Comment:
nit: We usually always use the latest editor's draft, so https://w3c.github.io/pointerevents/#pointerevent-interface.

Reply:
```suggestion
// https://w3c.github.io/pointerevents/#pointerevent-interface
```

Reply:
I'm still learning about the different spec versions. I assumed latest editor draft might also include new features like an alpha release. But I guess it only includes editorial changes which means it's perfectly safe to use as a reference for implementations?

Reply:
@eps1lon The latest editor's draft is the document that implementors use, as they are the latest versions of everything. For the same reason why we use the Living Standards for WHATWG specs, we use editor's draft for W3C specs (they are generally considered to be quite equivalent).

Reply:
This still doesn't seem to be fixed?

Reply:
Sorry I juggle different versions of it locally and it got lost. Is there an easier way to tell the test runner to only run on a subset?